components/org.wso2.carbon.identity.handler.event.account.lock/src/main/java/org/wso2/carbon/identity/handler/event/account/lock/AccountDisableHandler.java

```diff
@@ -386,6 +408,48 @@ private void setUserClaim(String claimName, String claimValue, UserStoreManager
         }
     }
 
+    private void publishPreAccountDisabledEvent(String accountDisabledEventName, Map<String, Object> map) throws
+            AccountLockException {
+
+        AccountUtil.publishEvent(accountDisabledEventName, AccountUtil.cloneMap(map));
+    }
+
+    private void publishPostAccountDisabledEvent(String accountDisabledEventName, Map<String, Object> map, boolean
+            isDisblePropertySuccessfullyModified) throws AccountLockException {
+
+        Map<String, Object> eventProperties = AccountUtil.cloneMap(map);
+        eventProperties.put(IdentityEventConstants.EventProperty.UPDATED_DISABLED_STATUS,
+                isDisblePropertySuccessfullyModified);
+        AccountUtil.publishEvent(accountDisabledEventName, eventProperties);
+    }
+
+    private static void createAuditMessage(String action, String target, JSONObject dataObject, String result) {
+
+        String loggedInUser = PrivilegedCarbonContext.getThreadLocalCarbonContext().getUsername();
+        if (StringUtils.isBlank(loggedInUser)) {
+            loggedInUser = AuditConstants.REGISTRY_SYSTEM_USERNAME;
+        }
+        String tenantDomain = PrivilegedCarbonContext.getThreadLocalCarbonContext().getTenantDomain();
+        loggedInUser = UserCoreUtil.addTenantDomainToEntry(loggedInUser, tenantDomain);
+        CarbonConstants.AUDIT_LOG.info(String.format(AuditConstants.AUDIT_MESSAGE, loggedInUser, action, target, dataObject, result));
+    }
```

Comment:
Let's move `createAuditMessage` to [1] as a common utils method to avoid code repetition in AccountDisableHandler and AccountLockHandler.

[1] https://github.com/wso2-extensions/identity-event-handler-account-lock/blob/master/components/org.wso2.carbon.identity.handler.event.account.lock/src/main/java/org/wso2/carbon/identity/handler/event/account/lock/util/AccountUtil.java